test/sharness/t0130-multinode.sh

```diff
@@ -87,6 +87,13 @@ test_expect_success "set up tcp testbed" '
 	iptb init -n 5 -p 0 -f --bootstrap=none
 '
 
+# test multiplex muxer
+export LIBP2P_MUX_PREFS="/mplex/6.7.0 /yamux/1.0.0 /spdy/3.1.0"
```

Comment:
just to be a bit more certain of what we're testing, lets remove yamux and spdy from this list.